toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -361,3 +367,27 @@ func loadOrDisableModules(modules imagecustomizerapi.Modules, imageChroot *safec
 
 	return nil
 }
+
+func addCustomizerRelease(imageChroot *safechroot.Chroot) error {
```

Comment:
This function seems simple enough that we can add UTs for it.

Check out `TestUpdateHostname` in `customizeutils_test.go`.